### DISCUSSION TITLE
How to generate lawyers.txt?

### DISCUSSION THREAD

**Discussion by ShubhamPatilsd** [supabase/supabase#2638] [Questions] [answered]
So this is not really a Supabase issue or anything. I was wondering how exactly was `lawyers.txt` was generated for supbase.io? I want to incorporate this in one of my projects and was wondering what tool was used to do this (if this was done manually, I'm impressed)

**Comment by inian** [comment, marked as answer]
Nothing too fancy Shubham
```
npm ls --parseable | xargs -L1 sh -c 'basename $1' dummy | sort | uniq > lawyers.txt
```

**Reply by ShubhamPatilsd** [reply]
Is it possible to do fancy styling with this like supabase?

**Comment by hash-bang** [comment]
Just in case anyone else ends up here hunting for a decent solution, I wrote up a NPM module to handle this - [@MomsFriendlyDevCo/Lawyers.txt](https://github.com/MomsFriendlyDevCo/lawyers.txt)
